sherpa/astro/ui/tests/test_astro_ui_utils_simulation.py

```diff
@@ -176,7 +178,9 @@ def test_fake_pha_basic(id, has_bkg, clean_astro_ui):
     # is summing the counts (to average over the randomness)
     # and then a simple check
     #
-    assert faked.counts.sum() > 200
+    assert (faked.counts.sum() > 200) and (faked.counts.sum() < 3000)
```

Comment:
tightened a few tests. I did not see random failures,  I think it's OK. If we see those tests fail for no reason and it becomes a problem we can always loosen the requirements again.

Reply:
Would it make sense to use an explicit seed and add the `reset_seed` fixture to reset the seed at the end of the test. Then we can make the tests much-more exact.

Reply:
I could do that if you prefer, but I also like to have the test run with a different random seed every time just to make sure we did not happen to catch some special case with the seed I choose. If these tests fail more than 1:1000 times, we can do that. For now, I think we are safe.

Reply:
Fixed seed for one test that I've seen fail and leave free for the others (that I have not seen fail ever).